### PR TITLE
Add uninstall kam crd with uninstall.sh

### PR DESCRIPTION
**Pull request by susanodom**

### CODE REVIEW

File: uninstall.sh [hunk 1]
```diff
@@ -40,4 +40,5 @@ fi
 
 kubectl delete -f ../operator/kappnav-delete-CR.yaml -n kappnav --now
 kubectl delete -f ../operator/kappnav-delete.yaml -n kappnav
-kubectl delete namespace kappnav
\ No newline at end of file
+kubectl delete -f ../operator/deploy/crds/actions_v1_kindactionmapping_crd.yaml -n kappnav
```
Comment (mrglavas) [inline uninstall.sh:43]: I don't think we've been uninstalling other CRDs. Certainly it was a conscious decision for the operator to install CRDs but never uninstall them. Deleting a CRD also causes all of the CRs for that kind to be deleted which may be harmful if some of those CRs were created/owned by another component/product (e.g. Appsody which might have its own KAM CR). Should double check whether this is something that we want to be doing.

**Review by mrglavas — CHANGES_REQUESTED** [review]
See comments.

File: uninstall.sh (hunk 1 above)
Comment (susanodom) [reply uninstall.sh:43]: Michael, good point - we still need to remove it if we remove kappnav namespace.  Yes, we need to talk more about what we want to do.

Comment (mrglavas) [reply uninstall.sh:43]: Not sure how that's tied to the kappnav namespace, but want to point out that the CRD doesn't have a namespace and CRs could be installed in other namespaces than kappnav.

Comment (susanodom) [reply uninstall.sh:43]: Yes - CRD is globally scoped - I am going to close this PR w/o merge as we do not need to remove the KAM CRD.

**Comment by susanodom** [comment]
Close the PR as the code change is not needed.

**Pull request closed without merging**